docs/get-started.md

```diff
@@ -233,34 +377,34 @@ You can create both tag and edge's schema:
 
 ```
 // create tags schema: players and teams:
-CREATE TAG player(name string, age int)
+nebula> CREATE TAG player(name string, age int)
 
-CREATE TAG team(name string)
+nebula> CREATE TAG team(name string)
 
 // create edges schema: players and teams:
-CREATE EDGE like(likeness int)
+nebula> CREATE EDGE like(likeness int)
 
-CREATE EDGE serve(start_year int, end_year int)
+nebula> CREATE EDGE serve(start_year int, end_year int)
 ```
 
 You could describe the schema created, for example:
 
 ```
-DESCRIBE TAG player
+nebula> DESCRIBE TAG player
 
-DESCRIBE EDGE serve
+nebula> DESCRIBE EDGE serve
 ```
 
 The insert sentences look like the following commands.
 
 ```
 // Insert some vertices: players and teams:
-INSERT VERTEX player(name, age) VALUES -8379929135833483044:("Amar'e Stoudemire", 36)
+nebula> INSERT VERTEX player(name, age) VALUES -8379929135833483044:("Amar'e Stoudemire", 36)
 
-INSERT VERTEX team(name) VALUES -9110170398241263635:("Magic")
+nebula> INSERT VERTEX team(name) VALUES -9110170398241263635:("Magic")
 
 // Insert some edges: likes and serves:
-INSERT EDGE like(likeness) VALUES -8379929135833483044 -> 6663720087669302163:(90)
+nebula> INSERT EDGE like(likeness) VALUES -8379929135833483044 -> 6663720087669302163:(90)
```

Comment:
According to the steps, there is no node whose vertexId is 6663720087669302163. When insert the edge, the users may get confused.

Reply:
> According to the steps, there is no node whose vertexId is 6663720087669302163. When insert the edge, the users may get confused.

right. I think it's better to simplify this example as following:

nebula> INSERT VERTEX player(name, age) VALUES 100:("Amar'e Stoudemire", 36)

nebula> INSERT VERTEX team(name) VALUES 200::("Magic")

nebula> INSERT EDGE like(likeness) VALUES 100 -> 200:(90)